docs/standard/frameworks.md

```diff
@@ -57,13 +58,54 @@ For each .NET 5.0 and later TFM, for example, `net5.0`, there are TFM variations
 | \<base-tfm>-watchos | net5.0-watchos |
 | \<base-tfm>-windows | net5.0-windows |
 
-You can also specify an optional OS version, for example, `net5.0-ios12.0`.
+The `net5.0` TFM only includes technologies that work cross-platform. Specifying an OS-specific TFM makes APIs that are specific to an operating system available to your app, for example, Windows Forms or iOS bindings. OS-specific TFMs also inherit every API available to the `net5.0` TFM.
 
-For more information about .NET 5 TFMs, see [Target framework names in .NET 5](https://github.com/dotnet/designs/blob/master/accepted/2020/net5/net5.md).
+To make your app portable across different platforms, you can target multiple OS-specific TFMs and add platform guards around OS-specific API calls using `#if` preprocessor directives.
+
+The following table shows the compatibility of the .NET 5 TFMs with TFMs for previous .NET versions.
+
+| TFM             | Compatible with                                            | Notes |
+|-----------------|------------------------------------------------------------|-|
+| net5.0          | net1..4 (with NU1701 warning)<br />netcoreapp1..3.1 (warning when WinForms or WPF is referenced)<br />netstandard1..2.1 | |
+| net5.0-android  | xamarin.android (plus everything else inherited from `net5.0`) | |
+| net5.0-ios      | xamarin.ios (plus everything else inherited from `net5.0`) | |
+| net5.0-macos    | xamarin.mac (plus everything else inherited from `net5.0`) | |
+| net5.0-tvos     | xamarin.tvos (plus everything else inherited from `net5.0`) | |
+| net5.0-watchos  | xamarin.watchos (plus everything else inherited from `net5.0`) | |
+| net5.0-windows  | netcoreapp1..3.1 (plus everything else inherited from `net5.0`) | Includes WinForms, WPF, and UWP APIs.<br />For information, see [Call Windows Runtime APIs in desktop apps](/windows/apps/desktop/modernize/desktop-to-uwp-enhance). |
+
+#### Suggested targets
+
+Use these guidelines to determine which TFM to use in your app:
+
+- Apps that are universal or portable to many platforms should target `net5.0`. This includes most libraries but also ASP.NET Core and Entity Framework.
```

Comment:
Nit: I would not use the term "universal" - sounds too much like "UWP". Also so far we've been using "portable" as the term for apps which can run on multiple platforms.